pages/index.tsx

```diff
@@ -59,19 +59,13 @@ export default function Home({restaurants}: HomeProps) {
 import * as DataService from '@/lib/DataService'
 import Restaurant from '@/models/Restaurant';
 
-/**
- * Loads data on the server side as the page loads.
- * @returns properties for the `index` page with the data loaded.
- */
 export async function getStaticProps() {
```

Comment:
I would re-add this deleted documentation!

Reply:
Maybe edit the `index` part to `user/[id]` so that the documentation is accurate!